packages/supertape/lib/cli.js

```diff
@@ -178,8 +180,8 @@ async function cli({argv, cwd, stdout, isStop}) {
     
     for (const file of files) {
         // always resolve before import for windows
-        const resolved = pathToFileURL(resolvePath(cwd, file));
-        promises.push(simpleImport(resolved));
+        const resolved = resolvePath(cwd, file);
+        promises.push(importOrRequire(resolved));
```

Comment:
So you want to add it to every loaded file so it was `imported` or `required`? Why not just use `ts-node` for this purpose 🤔?

Reply:
A better solution that simply checks the extension:

```js
if (extnamePath(file) === '.ts') {
  const resolved = resolvePath(cwd, file);
  promises.push(require(resolved));
} else {
  const resolved = pathToFileURL(resolvePath(cwd, file));
  promises.push(simpleImport(resolved));
}
```

Maybe it would be better to check that the extension does not contain `js`? In case somebody wants to use this with another superset language of JavaScript.